apps/openmw/mwworld/containerstore.cpp

```diff
@@ -420,6 +420,17 @@ int MWWorld::ContainerStore::remove(const std::string& itemId, int count, const
     return count - toRemove;
 }
 
+bool MWWorld::ContainerStore::hasVisibleItems() const
```

Comment:
And now it checks the truth of the fact that there *aren't* any visible items in the inventory. :P
The name is fine as it is but the returned values and the handling of the returned boolean should be inverted.